pkgs/os-specific/linux/kernel/generic.nix

```
@@ -6,6 +6,7 @@
 , gmp ? null
 , libmpc ? null
 , mpfr ? null
+, pahole ? null
```

```suggestion
, pahole
```
Please don't add new usages where packages can be overwritten to null

---

It is _technically_ optional here, so this is _technically_ correct. That said, the kernel build hard-depends on it, so it's probably a good idea to make it non-optional here too.

---

If something is optional that should not be represented with ``? null`` because that allows the package to not exist at all and is pretty good in hidding errors.